container-search/src/test/java/com/yahoo/prelude/cluster/ClusterSearcherTestCase.java

```diff
@@ -28,15 +27,7 @@
 import com.yahoo.vespa.config.search.DispatchConfig;
 import org.junit.jupiter.api.Test;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Iterator;
-import java.util.LinkedHashMap;
-import java.util.LinkedHashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
New IntelliJ again? :stuck_out_tongue_closed_eyes:

Reply:
Sigh, why can the default just be correct. #24800